server/neptune/workflows/internal/deploy/terraform/runner.go

```diff
@@ -36,6 +37,7 @@ type stateReceiver interface {
 type deployQueue interface {
 	GetOrderedMergedItems() []DeploymentInfo
 	GetQueuedRevisionsSummary() string
+	GetLockState() lock.LockState
```

Comment:
The reason I wanted to migrate LockState to its own package is so that the terraform package can also use this GetLockState function without running into cyclic package dependencies (queue package imports terraform package, but terraform package needs access to functions and structs defined in queue package).